parachains/runtimes/assets/statemine/src/weights/xcm/mod.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+mod pallet_xcm_benchmarks_fungible;
+mod pallet_xcm_benchmarks_generic;
+
+use crate::Runtime;
+use frame_support::weights::Weight;
+use sp_std::prelude::*;
+use xcm::{latest::prelude::*, DoubleEncoded};
+
+use pallet_xcm_benchmarks_fungible::WeightInfo as XcmFungibleWeight;
+use pallet_xcm_benchmarks_generic::WeightInfo as XcmGeneric;
+
+trait WeighMultiAssets {
+	fn weigh_multi_assets(&self, weight: Weight) -> Weight;
+}
+
+// TODO: This needs to be changed, not sure what to
+const MAX_ASSETS: u32 = 1;
```

Comment:
Not sure what to use here either. https://github.com/paritytech/polkadot/issues/4426 is still open.  
Maybe try 100?